develop/devguide/Connector/ImpactOfProtocolChanges/ImpactOfProtocolVersionChanges.md

```diff
@@ -27,7 +27,9 @@ It is important to assess if a major change will be needed as soon as possible t
 
 There are many protocol changes that can cause impact.
 
-Causing impact means that if you have an existing element using a previous version of this protocol, you cannot upgrade to the new version because something in your system will (or might) break depending on what specific features of the protocol or DataMiner are being used.
+The nature and severity of the impact will be documented via the connector XML [VersionHistory](xref:Manifest.VersionHistory) subtags as well as the actions to take to overcome those. This information will then be exposed both in the Catalog and in DataMiner Cube when a pre-existing element is updated.
```

Comment:
Something isn't quite right in this sentence. Do I interpret everything correctly if I change it like this?
```suggestion
The nature and severity of the impact will be documented via the connector XML [VersionHistory](xref:Manifest.VersionHistory) subtags along with the actions that need to be taken to overcome the impact. This information will then be exposed both in the Catalog and in DataMiner Cube when a pre-existing element is updated.
```

Reply:
correct!